azure/table_azure_mssql_managed_instance.go

```diff
@@ -479,3 +486,127 @@ func listMSSQLManagedInstanceVulnerabilityAssessments(ctx context.Context, d *pl
 
 	return managedInstanceVulnerabilityAssessments, nil
 }
+
+// If we return the API response directly, the output will not provide 
+// the properties of SecurityAlertPolicies
+func listMSSQLManagedInstanceSecurityAlertPolicies(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("listMSSQLManagedInstanceSecurityAlertPolicies")
+
+	managedInstance := h.Item.(sql.ManagedInstance)
+	resourceGroup := strings.Split(string(*managedInstance.ID), "/")[4]
+	managedInstanceName := *managedInstance.Name
+
+	session, err := GetNewSession(ctx, d, "MANAGEMENT")
+	if err != nil {
+		return nil, err
+	}
+	subscriptionID := session.SubscriptionID
+
+	client := sql.NewManagedServerSecurityAlertPoliciesClient(subscriptionID)
+	client.Authorizer = session.Authorizer
+
+	op, err := client.ListByInstance(ctx, resourceGroup, managedInstanceName)
+	if err != nil {
+		plugin.Logger(ctx).Error("listMSSQLManagedInstanceSecurityAlertPolicies", "list", err)
+		return nil, err
+	}
+
+	var managedInstanceSecurityAlertPolicies []map[string]interface{}
+
+	for _, i := range op.Values() {
+		managedInstanceSecurityAlertPolicy := make(map[string]interface{})
```

Comment:
Can we write a transform function which will map the ManagedInstanceSecurityAlertPolicy  properties instead of writing the same code again and again?